roles/zabbix_agent/tasks/api.yml

```diff
@@ -1,5 +1,6 @@
 ---
 - name: "API | Create host groups"
+  throttle: "{{ zabbix_api_throttle | int }}"
```

Comment:
```suggestion
```
We don't need throttle for everyone. I regularly use this role on tens of hosts and never experience any issues except when your Ansible controller is running on MacOS (see https://github.com/ansible-collections/community.zabbix/issues/1112)
If you want you can play with `forks` and `serial` on playbook level.

Reply:
I am indeed often running Ansible on mij Mac.
Thanks for the hint, I will try to find another solution.